ansible/files/admin_api_scripts/commence_walg_backup.sh

```diff
@@ -2,6 +2,6 @@
 
 set -euo pipefail
 
-WALG_SENTINEL_USER_DATA="{ \"backup_id\": $1, \"project_id\": $2 }" nohup wal-g backup-push /var/lib/postgresql/data --config /etc/wal-g/config.json --verify >> /var/log/wal-g/backup-push.log 2>&1 &
+WALG_SENTINEL_USER_DATA="{ \"project_id\": $1, \"backup_id\": $2 }" nohup wal-g backup-push /var/lib/postgresql/data --config /etc/wal-g/config.json --verify >> /var/log/wal-g/backup-push.log 2>&1 &
```

Comment:
How much data do we expect this script to spit out? Do we need to set up logrotate on this file?

Reply:
It's dependent on the size the database but for the smaller ones, it'll be approx 20 lines. For a large one (~15 GB) it takes up around 50 lines per full backup (incremental backups take approximately 20 lines). Logrotate has already been set up [here](https://github.com/supabase/postgres/blob/develop/ansible/files/logrotate_config/logrotate-walg.conf).